### PR TITLE
Add accept header

### PR DESCRIPTION
**Pull request by RickCraig**
Related to #21 

Adding the `Accept` header will hopefully fix the issue with receiving XML off the hooks endpoint.

### CODE REVIEW

File: lib/bigcommerce.js [hunk 1]
```diff
@@ -142,9 +142,20 @@ BigCommerce.prototype.authorise = function(query, cb) {
 };
 
 BigCommerce.prototype.createAPIRequest = function() {
+  var accept = 'application/json';
+  switch (this.config.responseType) {
+    case 'xml':
+      accept = 'text/xml';
+      break;
+    case 'json':
+      accept = 'application/json';
```
Comment (dlebech) [inline lib/bigcommerce.js]: This case seems redundant, given that `accept` is already set above.

File: lib/bigcommerce.js [hunk 2]
```diff
@@ -142,9 +142,20 @@ BigCommerce.prototype.authorise = function(query, cb) {
 };
 
 BigCommerce.prototype.createAPIRequest = function() {
+  var accept = 'application/json';
+  switch (this.config.responseType) {
+    case 'xml':
+      accept = 'text/xml';
+      break;
+    case 'json':
```
Comment (stefanosala) [inline lib/bigcommerce.js]: You don't need this since you already set `var accept = 'application/json';` above

**Comment by Esposito-NoFraud** [comment]
Any updates on this front? I was the one who found and reported the issue. Having a timeframe from y'all will help me timeframe this for my company as well.

**Comment by jmnsf** [comment]
@RickCraig ready to merge?

**Comment by RickCraig** [comment]
Sorry for the delay

**Pull request merged**